identifiers/country-do/provincial_districts.csv

```diff
@@ -0,0 +1,57 @@
+id,name

```

Comment:
I think we should split this file, there should be one for the admnistrative areas and a separate file for the electoral districts

Reply:
Not all provinces are divided into electoral districts. So, I added all the ed in the same file to avoid confusion. I can split the file if you think otherwise.

Reply:
Yes, same in the official website: https://www.camaradediputados.gob.do/.

Reply:
Having a consistent type for the electoral districts would be preferable. Perhaps `ed:at-large` would make sense so that they are distinct from potential future districts if redistricting subdivides some of these. Thoughts?

Reply:
That's fine with me - we do that in other jurisdictions that don't explicitly have a single, numbered-"1" district.